src/servers/grpc_server.cc

```diff
@@ -1021,6 +1021,20 @@ CommonHandler::SetUpAllRequests()
                   ->set_ns(ucnt);
             }
 
+            {
+              triton::common::TritonJson::Value cache_hit_json;
+              err = infer_stats_json.MemberAsObject("cache_hit", &cache_hit_json);
+              GOTO_IF_ERR(err, earlyexit);
+
+              err = cache_hit_json.MemberAsUInt("count", &ucnt);
+              GOTO_IF_ERR(err, earlyexit);
+              statistics->mutable_inference_stats()
+                  ->mutable_cache_hit()
+                  ->set_count(ucnt);
+              err = cache_hit_json.MemberAsUInt("ns", &ucnt);
+              GOTO_IF_ERR(err, earlyexit);
+              statistics->mutable_inference_stats()->mutable_cache_hit()->set_ns(ucnt);
```

Comment:
Shouldn't the similar change be applied to http_server?

Reply:
I believe the JSON returned by HTTP server is setup here: https://github.com/triton-inference-server/server/blob/rmccormick-cache-stats/src/core/tritonserver.cc#L2257-L2282

The GRPC server has to specially handle grabbing each field from the JSON.